proco/connection_statistics/models.py

```diff
@@ -127,6 +137,22 @@ def get_connectivity_status(self):
         else:
             return self.CONNECTIVITY_STATUSES.moderate
 
+    def get_coverage_statuses(self):
+        coverage_type = self.COVERAGE_TYPES.unknown
+        coverage_availability = None
+
+        if self.connectivity_type and self.connectivity_type.lower() != self.CONNECTIVITY_TYPES.unknown:
+            coverage_availability = True
+            if self.connectivity_type.lower() in (_[0] for _ in SchoolWeeklyStatus.COVERAGE_TYPES):
```

Comment:
not sure we will be able to find exact matches for our values (also, counting brazil have own values), so maybe it will make sense to update check into something like
```
if '2g' in self.connectivity_type.lower():
    ...
```